README.adoc

```diff
@@ -80,13 +80,13 @@ The list of available container images is sourced from the cluster's internal co
 
 To list the available components and associated container images for your cluster:
 
-. Log in to the {product-title} cluster with {odo-title}:
+. Log in to the cluster with `odo`: 
```

Comment:
This is OpenShift specific. One can't login to the k8s cluster.

Reply:
Yes, and no :-) 
Even Kubernetes cluster can have authentication but we currently don't support that for Kubernetes.

Reply:
Yeah. My comment was based on `odo login` always failing for me when I am working with minikube cluster. :smile:

Reply:
WDYT about: "Access your cluster. For OpenShift cluster, you can authenticate with `odo login`. 
Authentication with `odo login` for Kubernetes clusters is currently not supported."

Reply:
@boczkowska Makes sense to me. In my experience, `odo` commands kind of just work with kubernetes. Login is not required for it.

Reply:
@dharmit reworded.